CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ To learn more about our roadmap, we recommend reading [this document](ROADMAP.md
 
 - **General:** Add support to customize HPA name ([3057](https://github.com/kedacore/keda/issues/3057))
 - **General:** Basic setup for migrating e2e tests to Go. ([#2737](https://github.com/kedacore/keda/issues/2737))
+- **General:** Introduce activationThreshold/minMetricValue for all scalers ([#2800](https://github.com/kedacore/keda/issues/2800))
```

Comment:
Do we add it now already, do it when the last one is done, or list the ones that have this already?

Reply:
I think that listing all will be a huge noise tbh, I prefer to add it when the last is already done or now

Reply:
Fine by me!

_PS: Just to clarify, I meant 1 line that has all the scalers that are done, not a line per scaler - That would be insane haha._

Reply:
I have removed it, I want to add all of them before the next release. Currently, I'm going slower because I'm migrating e2e test if they are not already in golang but if I don't have enough time, I'll just add the feature and update the e2e test in ts